src/components/storage/InstallationMethod.jsx

```diff
@@ -120,45 +124,42 @@ const CustomFooter = ({ isFormDisabled, isReclaimSpaceCheckboxChecked, setStepNo
     }, [isNextClicked, goToNextStep, newPartitioning, partitioning.path]);
 
     const onNext = async ({ setIsFormDisabled }) => {
-        if (method === "MANUAL") {
-            setNewPartitioning(partitioning.path);
+        const part = await getNewPartitioning({
+            autopartScheme: defaultScheme,
+            currentPartitioning: partitioning,
+            devices,
+            method,
+            selectedDisks,
+            storageScenarioId,
+        });
+        setNewPartitioning(part);
```

Comment:
Seemed to me that in case of method MANUAL partitioning is passed to setNewPartitioning here, while before it was partitioning.path ?

Reply:
I think this is not longer relevant.